### PR TITLE
tribute 0.3.1 (new formula)

### PR DESCRIPTION
**Pull request by nicklockwood**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/tribute.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  license "MIT"
+  head "https://github.com/nicklockwood/Tribute.git", branch: "master"
+
+  bottle do
+    sha256 cellar: :any_skip_relocation, arm64_monterey: "bacd0dc8f488fd7909252467eaf16dcd339468857aea13a3643f7e0efd7715f8"
+    sha256 cellar: :any_skip_relocation, arm64_big_sur:  "7bc39a57d110f727993aa556cca5cba5549d536367bc12f25b56e25c4fdfa194"
+    sha256 cellar: :any_skip_relocation, monterey:       "739d230372dd797d5cf5a0586156068ecb5965878550a67b5ca56ccdf2799557"
+    sha256 cellar: :any_skip_relocation, big_sur:        "22080bdc75b03f5926d68fb248fc401d6a94876205ce03663a8950644b0e41e6"
+    sha256 cellar: :any_skip_relocation, catalina:       "d4cda88cf3bef7b18e960197b8db4826f234e3b3434ba902c3924c37b9040fe8"
+  end
```
Comment (carlocab) [inline Formula/tribute.rb:15]:
```suggestion
```

File: Formula/tribute.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  depends_on xcode: ["10.1", :build]
+  depends_on :macos
+
+  def install
+    xcodebuild "-arch", Hardware::CPU.arch,
+        "-project", "Tribute.xcodeproj",
+        "-scheme", "Tribute",
+        "-configuration", "Release",
+        "CODE_SIGN_IDENTITY=",
+        "SYMROOT=build", "OBJROOT=build"
```
Comment (carlocab) [inline Formula/tribute.rb:26]:
```suggestion
    xcodebuild "-arch", Hardware::CPU.arch,
               "-project", "Tribute.xcodeproj",
               "-scheme", "Tribute",
               "-configuration", "Release",
               "CODE_SIGN_IDENTITY=",
               "SYMROOT=build", "OBJROOT=build"
```

File: Formula/tribute.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    bin.install "build/Release/tribute"
+  end
+
+  test do
+    system "#{bin}/tribute", "help"
```
Comment (carlocab) [inline Formula/tribute.rb:31]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Comment (nicklockwood) [reply Formula/tribute.rb]: @carlocab how's this?

**Pull request closed without merging**